cypress/integration/unit_tests/color_function_util_test.js

```diff
@@ -42,8 +145,8 @@ describe('Unit tests for color function utility', () => {
     const maxMin = $('#max-min');
     expect(maxMin.is(':visible')).to.be.false;
 
-    const continuousPropertyPicker = $('#continuous-property-picker');
-    continuousPropertyPicker.val('wings').trigger('change');
+    const propertyPicker = $('#property-picker');
```

Comment:
nit: could inline this, right? Could also use Cypress .get() (here and elsewhere, since creating inside Cypress doc).

Reply:
Inlined. I would rather stick with current syntax as to not have to chain non-cypressing things to get them in the right order if that's alright with you.